### PR TITLE
SONARIAC-778 Rule S5332: ARM Using clear-text protocols is security-sensitive; Microsoft.Cdn/profiles/endpoints

### PR DESCRIPTION
**Pull request by jonas-wielage-sonarsource**

### CODE REVIEW

File: iac-extensions/arm/src/test/java/org/sonar/iac/arm/checks/LogRetentionCheckTest.java
Comment (jonas-wielage-sonarsource) [inline iac-extensions/arm/src/test/java/org/sonar/iac/arm/checks/LogRetentionCheckTest.java:1]: I deleted the line mentioning the schema in the json test files, so every issue line moved one. Same for the other test.

Comment (rudy-regazzoni-sonarsource) [reply iac-extensions/arm/src/test/java/org/sonar/iac/arm/checks/LogRetentionCheckTest.java:1]: I would avoid doing so: the line about schema is a mandatory property, and I'm currently adding it as it is required for some rules to detect the use case :sweat_smile: (there are different schemas and the rule must raise only on few of them)

**Review by rudy-regazzoni-sonarsource — APPROVED** [review]
LGTM! Please see my comment about adding back the `$schema` unless there is a concern I didn't get.

File: iac-extensions/arm/src/test/java/org/sonar/iac/arm/checks/LogRetentionCheckTest.java
Comment (jonas-wielage-sonarsource) [reply iac-extensions/arm/src/test/java/org/sonar/iac/arm/checks/LogRetentionCheckTest.java:1]: Summary of discussion: This should not be a mandatory attribute.

**Pull request merged**